src/Fixtures/LaravelDatabaseInterface.php

```diff
@@ -42,6 +42,12 @@ public function wipe($connection, $table)
      */
     public function insert($connection, $table, $rows)
     {
+        foreach ($rows as &$row) {
+            $row = array_map(function($value) {
+                return $value === 'CURDATE()' ? \DB::raw('CURDATE()') : $value;
+            }, $row);
+        }
```

Comment:
This is better done by using a php yaml file. eg:
```yaml
# users.php in fixtures directory
bob:
  name: Bob
  created_at: <?= Carbon\Carbon::now() ?>
```

otherwise this becomes a sea of `DB::raw()` calls for each special case